src/XrdXrootd/XrdXrootdConfigMon.cc

```diff
@@ -478,7 +482,7 @@ int XrdXrootdProtocol::xmongs(XrdOucStream &Config)
 
    int numgs = sizeof(gsObj)/sizeof(struct XrdXrootdGSReal::GSParms);
    int selAll = XROOTD_MON_CCM | XROOTD_MON_PFC | XROOTD_MON_TCPMO
-              | XROOTD_MON_TPC;
+              | XROOTD_MON_THROT | XROOTD_MON_TPC;
```

Comment:
I see you nicely added throttle to the comments for XrdXrootdProtocol::xmon, could you do the same for XrdXrootdProtocol::xmongs? It's a one liner.

Reply:
Done in the latest rebase.